From: Babu Moger <babu.moger@netapp.com>
Date: Mon, 14 May 2012 13:32:06 +0200
Subject: scsi_dh_rdac: Add new NetApp IDs
References: bnc#757077
Patch-Mainline: n/a

This patch adds a new netapp vendor and product ids to rdac handler.

Signed-off-by: Babu Moger <babu.moger@netapp.com>
Acked-by: Hannes Reinecke <hare@suse.de>

diff --git a/drivers/scsi/device_handler/scsi_dh_rdac.c b/drivers/scsi/device_handler/scsi_dh_rdac.c
index 4d2f211..e3e8379 100644
--- a/drivers/scsi/device_handler/scsi_dh_rdac.c
+++ b/drivers/scsi/device_handler/scsi_dh_rdac.c
@@ -837,6 +837,7 @@ static const struct scsi_dh_devlist rdac_dev_list[] = {
 	{"DELL", "MD32xxi"},
 	{"DELL", "MD36xxi"},
 	{"DELL", "MD36xxf"},
+	{"NETAPP", "INF-01-00"},
 	{"LSI", "INF-01-00"},
 	{"ENGENIO", "INF-01-00"},
 	{"STK", "FLEXLINE 380"},
